avm/res/databricks/workspace/tests/e2e/waf-aligned/main.test.bicep

```diff
@@ -147,6 +148,21 @@ module testDeployment '../../../main.bicep' = [
       managedResourceGroupResourceId: '${subscription().id}/resourceGroups/rg-${resourceGroupName}-managed'
       requireInfrastructureEncryption: true
       vnetAddressPrefix: '10.100'
+      privateStorageAccount: 'Enabled'
+      accessConnectorId: nestedDependencies.outputs.accessConnectorId
```

Comment:
Not sure if this accessConnectorID should be accessConnectResourceID? Just checking

Reply:
You're right